### ISSUE TITLE
Project Status

### ISSUE THREAD

**Issue by poka-IT** [leonardocustodio/polkadart#433]
Hello everyone!

First of all, thank you very much to everyone who contributes to this valuable project.

I hope this will allow me to switch from the very limiting headless webview polkawallet-sdk to this full dart library for our [Ğecko app](https://git.duniter.org/clients/gecko).

I am part of the Duniter project team, and we would like to know the maintenance status of this library before deciding to overhaul the code of our Flutter apps.

Are these milestones up to date? https://grants.web3.foundation/applications/Polkadart

If not, what can we rely on?
We are ready to contribute to this library based on the features we need for our tools, but we want to ensure that this project is built to last.

**In your opinion, is this library mature enough to be used in production?**

Considering that our needs are limited to:

- **Connecting to a node ([Duniter v2s](https://git.duniter.org/nodes/rust/duniter-v2s), Blockchain monochain based on the latest version of Substrate) reactively from a list of wss endpoints.**
- **Retrieving any value from the onchain storage (as well as runtime constants), including multi queries**
- **Subscribing to any of these values**
- **Executing any extrinsic, and tracking its status live, including batch**

Thank you in advance for your clarification.

**Comment by leonardocustodio** [comment]
Hello @poka-IT ,

We have just finished the last milestone meaning every promise we made is now available within the lib. We already have people using in production with very few errors reported. We are confident that the lib is mature enough to be used in production.

All the points you wrote are supported. There are a few things that don't have that much helpers to reduce the boilerplate amounht needed but everything is supported. We hope in the future to start making helpers to reduce the boilerplates.
